docs/en/stack/ml/df-analytics/ml-dfanalytics-evaluate.asciidoc

```diff
@@ -118,14 +120,32 @@ the {reganalysis} model is performing.
 [[ml-dfanalytics-r-sqared]]
 === R-squared
 
-Another evaluation metrics for {reganalysis} is R-squared (R^2^). It represents 
+Another evaluation metric for {reganalysis} is R-squared (R^2^). It represents 
 the goodness of fit and measures how much of the variation in the data the 
 predictions are able to explain. The value of R^2^ are less than or equal to 1, 
 where 1 indicates that the predictions and true values are equal. A value of 0 
 is obtained when all the predictions are set to the mean of the true values. A 
 value of 0.5 for R^2^ would indicate that, the predictions are 1 - 0.5^(1/2)^ 
 (about 30%) closer to true values than their mean.
 
+[[ml-dfanalytics-huber]]
+=== Pseudo-Huber loss
+
+Pseudo-Huber loss metric behaves as mean absolute error (MAE) for errors larger 
```

Comment:
Does it make sense to add a hyperlink to the Wikipedia description of the pseudo-Huber loss?

Reply:
Sure, I've added the link in the latest commit. Thanks!